// @storybook/addon-vitest/dist/_node-chunks/chunk-T4XS23FV.js
import CJS_COMPAT_NODE_URL_m2yzxg9bz2c from 'node:url';
import CJS_COMPAT_NODE_PATH_m2yzxg9bz2c from 'node:path';
import CJS_COMPAT_NODE_MODULE_m2yzxg9bz2c from "node:module";

var __filename = CJS_COMPAT_NODE_URL_m2yzxg9bz2c.fileURLToPath(import.meta.url);
var __dirname = CJS_COMPAT_NODE_PATH_m2yzxg9bz2c.dirname(__filename);
var require = CJS_COMPAT_NODE_MODULE_m2yzxg9bz2c.createRequire(import.meta.url);

// ------------------------------------------------------------
// end of CJS compatibility banner, injected by Storybook's esbuild configuration
// ------------------------------------------------------------
import {
  require_picocolors
} from "./chunk-P7LXRNSZ.js";
import {
  ADDON_ID2 as ADDON_ID
} from "./chunk-PDGQQL6V.js";
import {
  __toESM
} from "./chunk-5MGU7JRW.js";

// src/logger.ts
var import_picocolors = __toESM(require_picocolors(), 1);
import { logger } from "storybook/internal/node-logger";
var log = (message) => {
  logger.log(
    `${import_picocolors.default.magenta(ADDON_ID)}: ${message.toString().replaceAll(/(│\n|│  )/g, "").trim()}`
  );
};

// ../../../node_modules/es-toolkit/dist/function/noop.mjs
function noop() {
}

// src/utils.ts
function errorToErrorLike(error) {
  return {
    message: error.message,
    name: error.name,
    // avoid duplicating the error message in the stack trace
    stack: error.message + " " + error.stack?.replace(error.message, ""),
    cause: error.cause && error.cause instanceof Error ? errorToErrorLike(error.cause) : void 0
  };
}

export {
  noop,
  log,
  errorToErrorLike
};
