// storybook/dist/server-errors.js
import CJS_COMPAT_NODE_URL_07ga0het4puh from 'node:url';
import CJS_COMPAT_NODE_PATH_07ga0het4puh from 'node:path';
import CJS_COMPAT_NODE_MODULE_07ga0het4puh from "node:module";

var __filename = CJS_COMPAT_NODE_URL_07ga0het4puh.fileURLToPath(import.meta.url);
var __dirname = CJS_COMPAT_NODE_PATH_07ga0het4puh.dirname(__filename);
var require = CJS_COMPAT_NODE_MODULE_07ga0het4puh.createRequire(import.meta.url);

// ------------------------------------------------------------
// end of CJS compatibility banner, injected by Storybook's esbuild configuration
// ------------------------------------------------------------
import {
  AddonVitestPostinstallConfigUpdateError,
  AddonVitestPostinstallError,
  AddonVitestPostinstallFailedAddonA11yError,
  AddonVitestPostinstallPrerequisiteCheckError,
  AddonVitestPostinstallWorkspaceUpdateError,
  AngularLegacyBuildOptionsError,
  AutomigrateError,
  Category,
  CommonJsConfigNotSupportedError,
  ConflictingStaticDirConfigError,
  CouldNotEvaluateFrameworkError,
  CriticalPresetLoadError,
  FindPackageVersionsError,
  GenerateNewProjectOnInitError,
  GoogleFontsDownloadError,
  GoogleFontsLoadingError,
  IncompatiblePostCssConfigError,
  InvalidFrameworkNameError,
  InvalidStoriesEntryError,
  MainFileEvaluationError,
  MainFileMissingError,
  MinimumReleaseAgeHandledError,
  MissingAngularJsonError,
  MissingBuilderError,
  MissingFrameworkFieldError,
  NoMatchingExportError,
  NoStatsForViteDevError,
  NxProjectDetectedError,
  SavingGlobalSettingsFileError,
  StatusTypeIdMismatchError,
  StorybookError,
  SvelteViteWithSvelteKitError,
  UpgradeStorybookToLowerVersionError,
  UpgradeStorybookUnknownCurrentVersionError,
  ViteModuleGraphSubscriptionError,
  WebpackCompilationError,
  WebpackInvocationError,
  WebpackMissingStatsError
} from "./_node-chunks/chunk-YM6JOEYD.js";
import "./_node-chunks/chunk-QT24SDPV.js";
import "./_node-chunks/chunk-7IV5W6W4.js";
import "./_node-chunks/chunk-N2NC4ZUG.js";
export {
  AddonVitestPostinstallConfigUpdateError,
  AddonVitestPostinstallError,
  AddonVitestPostinstallFailedAddonA11yError,
  AddonVitestPostinstallPrerequisiteCheckError,
  AddonVitestPostinstallWorkspaceUpdateError,
  AngularLegacyBuildOptionsError,
  AutomigrateError,
  Category,
  CommonJsConfigNotSupportedError,
  ConflictingStaticDirConfigError,
  CouldNotEvaluateFrameworkError,
  CriticalPresetLoadError,
  FindPackageVersionsError,
  GenerateNewProjectOnInitError,
  GoogleFontsDownloadError,
  GoogleFontsLoadingError,
  IncompatiblePostCssConfigError,
  InvalidFrameworkNameError,
  InvalidStoriesEntryError,
  MainFileEvaluationError,
  MainFileMissingError,
  MinimumReleaseAgeHandledError,
  MissingAngularJsonError,
  MissingBuilderError,
  MissingFrameworkFieldError,
  NoMatchingExportError,
  NoStatsForViteDevError,
  NxProjectDetectedError,
  SavingGlobalSettingsFileError,
  StatusTypeIdMismatchError,
  StorybookError,
  SvelteViteWithSvelteKitError,
  UpgradeStorybookToLowerVersionError,
  UpgradeStorybookUnknownCurrentVersionError,
  ViteModuleGraphSubscriptionError,
  WebpackCompilationError,
  WebpackInvocationError,
  WebpackMissingStatsError
};
